lib/ripper-tags/emacs_formatter.rb

```diff
@@ -55,6 +72,8 @@ def start_file_section(filename, io)
 
     def flush_file_section(out)
       if @section_io
+        save_section(@current_file) { |tag| record(@section_io, tag) }
```

Comment:
Why was this edit to `flush_file_section` necessary?

Reply:
To dump old symbols from the file we've just processed. Otherwise we'd have to dump them at the very end, potentially creating more than one section per sourcefile.

Reply:
This may lead to stale symbols being included (when we remove a declaration from the file, and rerun with `--append` - the old symbol will still be in the tagsfile). Not sure how to handle that yet.